test/inductor/test_aot_inductor_utils.py

```diff
@@ -19,6 +20,13 @@
 from torch.utils import _pytree as pytree
 
 
+pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(pytorch_test_dir)
+from inductor.test_torchinductor import (  # @manual=fbcode//caffe2/test/inductor:test_inductor-library
+    clone_preserve_strides,
+)
```

Comment:
This feels wrong. If utility is generally useful (looks like it is), it should be moved to `torch.testing.module`

Reply:
`check_model` is for AOTI test only, so I think keeping it here is fine.

I do agree on your `sys.path` comment.